pkgs/desktops/lxqt/liblxqt/default.nix

```diff
@@ -41,6 +41,11 @@ mkDerivation rec {
 
   postPatch = ''
     sed -i "s|\''${POLKITQT-1_POLICY_FILES_INSTALL_DIR}|''${out}/share/polkit-1/actions|" CMakeLists.txt
+    # convert name of wrapped binary, e.g. .lxqt-whatever-wrapped to the original name, e.g. lxqt-whatever so binaries can find their resources
+    substituteInPlace lxqtapplication.cpp \
+        --replace 'QFileInfo(applicationFilePath()).fileName()' 'QFileInfo(applicationFilePath()).fileName().mid(1).chopped(8)'
+    substituteInPlace lxqttranslator.cpp \
```

Comment:
Since we have automatic updates it would be better to make this a patch to make it fail more obviously if it no longer matches.